src/docs/user/ProActiveUserGuide.html

```diff
@@ -0,0 +1,3480 @@
+<!DOCTYPE html>
```

Comment:
Why did you add ProActiveUserGuide ? If it is an error please remove it. Compiled file should not be added on github.